### ISSUE TITLE
JS Interactions and animations

### ISSUE THREAD

**Issue by ziogio98** [soflyy/oxygen-bugs-and-features#264]
**Describe the feature you'd like to see included in Oxygen.**
I'd love to have interactions and advanced animation in Oxygen. This is obviously not urgent, not 100% needed, but it's one of the best features of Webflow (ecommerce is another one, but no need to tell you about that ahah).

**What are the use cases for this feature?**
It's basically a vanity feature, but it's great to create complex designs. Web designers are using animations most often than ever, and they're not always scroll animation. They may be on click animation, on load animation, or even horizontal animation (which are absolutely not user friendly, though). 

Basically, you could use a mix of the modal element + animation theory. Meaning something as simple as "select which type of triggers you like and what action should happen then" :) 

**Examples of this feature or functionality.**
Webflow Interaction is the only example.

*If another product has done it well, provide examples here.*
Webflow has done it beautifully.

**Comment by tnomads** [comment]
would be an amazing feature

**Comment by agencia-knut** [comment]
That would be an amazing new feature! 🥇 

**Webflow**, in one hand, and **Divi** as a Wordpress builder, in the other hand, are strong competitors of Oxygen and both offer advanced interaction and animation options. I'd love to hear that Oxygen will also have this useful feature in a near future. Will be a great addition for web designers!

Thanks!! :)

**Comment by kain-kuin** [comment]
I'm using Webflow and interaction/animation feature that it have is amazing! Custom pop-ups or any mega menu on top/left by click/hover — a lot you can imagine is possible to create in webflow easy (yep — easy). And it all done all the time with the same elements and interaction commands (choose trigger element, choose condition (like hover/click/scroll etc.), choose target element, choose CSS values, set easing — that's it) — so no need to learn different interface/fields on different custom element, you can do all with basic divs/links etc.

**Comment by farhanmunim** [comment]
+1
